modules/knative-services-connectivity.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * serverless/getting-started-knative-services.adoc
+
+[id="knative-services-connectivity_{context}"]
+= Connecting Knative Services to existing Kubernetes deployments
+
+Knative Services can call a Kubernetes deployment in any namespace, provided that there are no existing additional network barriers.
+
+A Kubernetes deployment can call a Knative Service if:
+
+* The Kubernetes deployment is in the same namespace as the target Knative Service.
+* The Kubernetes deployment is in a namespace that has manually been added to the ServiceMeshMemberRoll in `knative-serving-ingress`.
+* The Kubernetes deployment uses the target Knative Service's public URL.
+
+[NOTE]
+====
+Knative Services can be accessed by using a public URL by default. The target Knative Service must not be configured as cluster-local only so that the Kubernetes deployment can access the Service using the public URL.
```

Comment:
This note only applies to the last bullet above. It only applies if the Kubernetes deployment is not in the same ServiceMeshMemberRoll as the Knative Service (which is provided by the first two bullets above).

If a K8s deployment is in the same namespace as the ksvc or in the same smmr, a cluster-local service works.

Reply:
Fixed formatting so this is part of the bullet